crates/wallet/src/wallet/changeset.rs

```diff
@@ -104,14 +104,16 @@ impl ChangeSet {
             .query_row([], |row| {
                 Ok((
                     row.get::<_, Impl<Descriptor<DescriptorPublicKey>>>("descriptor")?,
```

Comment:
```suggestion
                    row.get::<_, Option<Impl<Descriptor<DescriptorPublicKey>>>>("descriptor")?,
```

I'm thinking it also makes sense to NOT error when we don't find an external descriptor. If we don't have an external descriptor, the problem isn't with our persistence implementation (the problem is with the caller not persisting an external descriptor).

We already catch the error if we have a missing descriptor in `Wallet::load_with_params`. This is the appropriate place to catch this error, since this is irrelevant to our persistence impl.

Reply:
Do the same for `network` as well?

Reply:
> Do the same for `network` as well?

I think so, yes!